chord_metadata_service/mohpackets/docs/schema_manual.yml

```
@@ -454,6 +495,31 @@ components:
           - type: integer
```

Same comment as above
```suggestion
          - type: number
```